src/include/execution/exec/execution_settings.h

```diff
@@ -27,7 +27,7 @@ class EXPORT ExecutionSettings {
   }
 
   /** @return True if parallel query execution is enabled. */
-  constexpr bool GetIsParallelQueryExecution() const { return is_parallel_query_execution_; }
+  constexpr bool GetIsParallelQueryExecutionEnabled() const { return is_parallel_query_execution_; }
```

Comment:
Nit: rename to `is_parallel_execution_enabled_` ? As written, it's a little unclear, though function name clears it up.

Reply:
Renamed